Tests/Runtime/Unity.WebRTC.RuntimeTests.asmdef

```diff
@@ -1,13 +1,13 @@
 {
     "name": "Unity.WebRTC.RuntimeTests",
+    "rootNamespace": "",
     "references": [
         "Unity.WebRTC",
         "UnityEngine.TestRunner",
         "UnityEditor.TestRunner"
     ],
     "includePlatforms": [
         "Android",
-        "Editor",
```

Comment:
This line is needed to run unit-test on Editor.

Reply:
Could you merge this commit into your branch?
https://github.com/Unity-Technologies/com.unity.webrtc/commit/187d9f324a93fd21c1688900b5782d74be59feef

Reply:
These errors occurred when running runtime tests.
![image](https://user-images.githubusercontent.com/1132081/134114638-8ee2387a-b629-4ff6-884e-69e52f28a4e5.png)